components/user/card-seekbar.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="text-[0.75rem] mt-3">
+      <div class="bar h-1 bg-neutral-700 rounded-xl">
+        <div class="progress h-1 bg-primary rounded-xl bg-white" :style="{ width: `${percentage}%` }"></div>
+      </div>
+      <div class="flex justify-between items-start">
+        <div class="mb-0 mt-0.5 py-0 text-xs">{{ currentPosition }}</div>
+        <div class="mb-0 mt-0.5 py-0 text-xs">{{ songDuration }}</div>
+      </div>
+    </div>
+</template>
+  
+<script setup lang="ts">
+const props = defineProps<{
+    startTimeMs: number, 
+    endTimeMs: number
+}>();
+
+const currentDateTime = ref(new Date());
+
+onMounted(() => {
+    const interval = setInterval(() => currentDateTime.value = new Date(), 1000);
+    onUnmounted(() => clearInterval(interval));
+});
```

Comment:
You may use [`useNow`](https://vueuse.org/core/useNow/) instead, it'll be cleaner and overall a single line of code.

Reply:
Thanks a bunch for the tip!